doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+// Custom scripts for the Spyder docs site
+
+
+;(function () {
+
+    'use strict';
+
+    // Set the active image for the tour based on the element's class
+    function setActiveTourImage(activeElement) {
+        var activeClass = "tour-screenshot-active";
+        var classNames = activeElement.node.className.baseVal.split(" ");
+        for (var i = 0; i < classNames.length; i++) {
+            var imageToActivate = document.getElementById(classNames[i]);
+            if (imageToActivate) break;
+        };
+        var imageToDeactivate = document.getElementsByClassName(activeClass)[0];
+        imageToDeactivate.classList.remove(activeClass);
+        imageToActivate.classList.add(activeClass);
+    };
+
+    // Interactive tour driver
+    var driver = new Driver({
+        animate: false,
+        opacity: 0.1,
+        padding: 0,
+        allowClose: false,
+        onHighlightStarted: setActiveTourImage,
+    });
+
+    // Event handler to start tour
+    function startTour() {
+        driver.start();
+    };
+
+    // Interactive tour of Spyder for Quickstart using Driver
+    function setupTourDriver() {
+        driver.defineSteps([
+          {
+            element: "#introduction-rect",
+            popover: {
+              title: "Spyder",
+              description: 'Spyder is a powerful scientific IDE for Python. We will guide you through some of its most important features.',
+              position: "right",
+            },
+          },
+          {
+            element: "#toolbar-rect",
+            popover: {
+              title: "Toolbar",
+              description: 'The toolbar allows you to quickly access some of the most common commands in Spyder.',
+              position: "bottom",
+            },
+          },
+          {
+            element: "#statusbar-rect",
+            popover: {
+              title: "Status Bar",
+              description: 'The status bar shows your current Python environment, git branch, memory usage and various attributes of the currently active file.',
+              position: "top",
+            },
+          },
+          {
+            element: "#options-menu-rect",
+            popover: {
+              title: "Options Menu",
+              description: 'You can display the options menu by clicking the "hamburger" icon at the top right of the pane. It contains useful settings and actions relevant to the pane.',
+              position: "right",
+            },
+          },
+          {
+            element: "#context-menu-rect",
+            popover: {
+              title: "Context Menu",
+              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element you are hovering over.',
```

Comment:
```suggestion
              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element you are inspecting',
```

I'm not convinced `inspecting` is the right word here, but `hovering` has a technical meaning in GUIs (just passing the mouse over an element), which is wrong in this context.

Reply:
```suggestion
              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element under your cursor.',
```